os/mm/mm_heap/mm_mallinfo.c

```diff
@@ -64,6 +64,19 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
+#define MM_PREV_NODE_SIZE(x)	((x)->preceding & ~MM_ALLOC_BIT)
+#define IS_ALLOCATED_NODE(x)	((x)->preceding & MM_ALLOC_BIT)
+#define IS_FREE_NODE(x)		(!IS_ALLOCATED_NODE(x))
```

Comment:
Could you align these MACRO definitions please?

Reply:
It is already aligned using tabs and looking find in vim editor. 
Not sure why the issue here. Will try to align again.